tips/urls.py

```diff
@@ -4,8 +4,11 @@
 
 app_name = 'tips'
 urlpatterns = [
-    path('', views.board, name='board'),
+    path('', views.index, name='index'),
     path('add_tip/', views.add_tip, name='add_tip'),
     path('edit_tip/<int:tip_id>/', views.edit_tip, name='edit_tip'),
-    path('delete_tip/<int:tip_id>/', views.delete_tip, name='delete_tip'),
+    path('category/<str:category>/', views.filter_tip_by_category, name='category'),
+    path('filtered_by_date/<str:date>/', views.filter_tip_by_date, name='filtered_by_date'),
```

Comment:
the filtering should happen in the same url just by using a django form to send the view function the date to get the filtered tips back see Liors PR https://github.com/redhat-beyond/Xpense/pull/127/files where he adds the year filtering to the expenses in the mine page

Reply:
I'm sorry I have tryed to do it with forms but things did't work as exepted I think mayb we should leave it like that since the demo is tommorow and I really think it sould be merged right now